src/systems/sprites/sprite.frag

```diff
@@ -0,0 +1,49 @@
+#version 300 es
```

Comment:
I think this needs to be converted to GLSL ES 2.0 for compatibility with WebGL 1.0. I'd test on iOS to be certain

Reply:
Yeah sadly we should write all our shaders such that they run on WebGL1, and then rewrite them at runtime to work on WebGL2 when we are using it (this is what we are doing now for all our other shaders). You will also need to make sure multiview is accounted for (likely just by letting the shader code get patched at runtime like we are doing for our other shaders)